src/AssetsPage/index.jsx

```diff
@@ -20,8 +20,7 @@ class AssetsPage extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (prevProps.assetsParameters.assetTypes !== this.props.assetsParameters.assetTypes) {
-      // if filters changed, update the assetsList
+    if (prevProps.assetsParameters !== this.props.assetsParameters) {
```

Comment:
I've changed this to be a condition on the assetsParameters changing at all, since we want to include changes not just to the assetTypes but also to the sort state.